src/client/pages/OfferNew/Checkout/UpsellCard.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+
+import { Button } from '../../../components/buttons'
+
+const UpsellCardRoot = styled.div`
+  background-color: ${colorsV3.gray200};
+  border-radius: 8px;
+  padding: 1.5em;
+  margin-bottom: 1rem;
+
+  & > *:not(:last-child) {
+    margin-bottom: 1rem;
+  }
+`
+
+const Title = styled.h1`
+  margin: 0;
+  font-size: 1.25rem;
```

Comment:
I see this is the size in design. not sure if I have the latest version or not. 
```suggestion
  font-size: 1.5rem;
  line-height: 1.33
```